src/components/core.ts

```diff
@@ -167,15 +167,19 @@ export default class Core {
      */
     this.config.minHeight = this.config.minHeight !== undefined ? this.config.minHeight : 300;
 
+    const defaultBlockToolData: DefaultBlockToolData = {
+      text: '',
```

Comment:
I think it is not a solution because the `text` is still hardcoded

Reply:
Even `text` data is hard-coded, it can be controled in the constructor of customized default block tool. 
See: https://github.com/hata6502/editorjs-examples/blob/main/examples/ExampleDefaultBlockTool.ts#L28

Reply:
I'd say that's OK as temporary solution